AirCasting/SessionViews/SessionCartViewModel/DefaultDeleteSessionViewModel.swift

```diff
@@ -96,7 +96,7 @@ class DefaultDeleteSessionViewModel: DeleteSessionViewModel {
     private func processStreamDeleting() {
         self.measurementStreamStorage.accessStorage { [self] storage in
             guard let sessionToPass = try? storage.getExistingSession(with: session.uuid) else { return }
-            streamRemover.updateSession(session: sessionToPass) { result in
+            streamRemover.updateSession(session: sessionToPass) { [self] result in
```

Comment:
note that this doesn't change anything in practice. The block still captures `self` strongly